deploy/deploy-webapp.ps1

```diff
@@ -24,10 +24,10 @@ param(
     # Client application id
     $ApplicationClientId,
 
-    [Parameter(Mandatory=$false)]
+    [Parameter(Mandatory = $false)]
     [string]
     # Authority for client applications that are not configured as multi-tenant.
-    $Authority="https://login.microsoftonline.com/common"
+    $Authority = "https://login.microsoftonline.com/common"
```

Comment:
There is some weirdness here -- the backend expects instance + tenant ID, while the frontend expects authority. However the values need to match. This could be confusing for the user. Perhaps change to make them more consistent (e.g. both use instance + tenant id)?

Reply:
What's the format expected from the client side?